reviews/utils/review_utils.py

```diff
@@ -482,19 +482,36 @@ def auto_review(proposal: Proposal):
         for task in Task.objects.filter(session__study=study):
             reasons.extend(auto_review_task(study, task))
 
-        if study.stressful in [YesNoDoubt.YES, YesNoDoubt.DOUBT]:
+        if study.knowledge_security in [YesNoDoubt.YES, YesNoDoubt.DOUBT]:
             reasons.append(
                 _(
-                    "De onderzoeker geeft aan dat (of twijfelt erover of) het onderzoek op onderdelen of \
-als geheel zodanig belastend is dat deze ondanks de verkregen informed consent vragen zou kunnen oproepen."
+                    "De onderzoeker geeft aan dat er mogelijk kwesties zijn rondom \
+                    kennisveiligheid."
+                )
+            )
+
+        if study.researcher_risk in [YesNoDoubt.YES, YesNoDoubt.DOUBT]:
+            reasons.append(
+                _(
+                    "De onderzoeker geeft aan dat er mogelijk kwesties zijn \
```

Comment:
If we end up keeping these fields in the study model, it might be nice to add the trajectory number in this string.

f"De onderzoeker geeft aan dat er mogelijk kwesties zijn rondom de veiligheid van de betrokken onderzoekers in traject {{study.order}}."


Right now, if a proposal has two trajectories and both have trajectory risk, this reason will just show up twice identically.

Also please fix the backslash whitespace issue here as well.